app/streams/publishing_api/message_handler.rb

```diff
@@ -7,39 +7,61 @@ def self.process(*args)
 
   def initialize(message)
     @message = message
-
-    @new_item = PublishingAPI::MessageAdapter.to_dimension_item(message)
-    @old_item = Dimensions::Item.find_by(base_path: new_item.base_path, latest: true)
   end
 
   def process
-    return unless new_item.older_than?(old_item)
-
-    if is_links_update?
-      grow_dimension! if links_have_changed?
-    else
-      grow_dimension!
+    items.each do |item|
```

Comment:
I would rename `items` to `items_handlers` as the Item does not have this behaviour.